src/chat-manager.js

```diff
@@ -93,4 +111,27 @@ export class ChatManager {
   disconnect() {
     if (this.currentUser) this.currentUser.disconnect()
   }
+
+  disablePushNotifications() {
+    try {
+      return this.beamsInstanceInitFn()
+        .then(beamsClient => {
+          return beamsClient.stop()
+        })
+        .catch(err => {
+          this.logger.warn(
+            `Chatkit error when disabling push notifications`,
```

Comment:
Minor: backticks aren't needed here if you're not doing string interpolation.  Though maybe you intended to interpolate the error?